ios/Exponent/Versioned/Modules/Api/EXNotifications.m

```diff
@@ -50,6 +50,7 @@ - (instancetype)initWithExperienceId:(NSString *)experienceId kernelServiceDeleg
 }
 
 RCT_REMAP_METHOD(getDevicePushTokenAsync,
+                 config:(NSDictionary *)config
                  getDevicePushTokenAsyncWithResolver:(RCTPromiseResolveBlock)resolve
```

Comment:
This will create an objective-c method called `config:getDevicePushTokenAsyncWithResolver:rejecter:` which I believe is not what you intend to do.

Probably want `getDevicePushTokenWithConfig: (__unused NSDictionary *)config resolver: ...`

Reply:
@terribleben I've attempted to fix this with the latest push, but honestly I've never tried writing Objective-C before so I'm probably wrong. I really appreciate your patience!

Reply:
Thanks - looks good with your recent change!